babel.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = function (api) {
 				},
 			},
 		],
+		["@babel/plugin-proposal-private-property-in-object", { loose: false }],
```

Comment:
try saying that with a mouth full of crisps

Reply:
Tbf - I'm not entirely sure what's going on with that other than it spams errors if youi don't have it.